services/app-api/src/resolvers/unlockHealthPlanPackage.test.ts

```diff
@@ -575,13 +576,21 @@ describe('unlockHealthPlanPackage', () => {
         const rateName = generateRateName(sub, sub.rateInfos[0], programs)
 
         // email subject line is correct for CMS email
-        expect(mockEmailer.sendEmail).toHaveBeenCalledWith(
+        expect(mockEmailer.sendEmail).toHaveBeenNthCalledWith(
+            4,
             expect.objectContaining({
                 subject: expect.stringContaining(`${name} was unlocked`),
                 //Rate name should have defaulted back to using package programs to generate name
                 bodyHTML: expect.stringContaining(rateName),
             })
         )
+        // expect(mockEmailer.sendEmail).toHaveBeenCalledWith(
+        //     expect.objectContaining({
```

Comment:
nit: i'm guessing this meant to be commented in or else removed

Reply:
thx